pom.xml

```diff
@@ -102,8 +102,8 @@
 				<artifactId>maven-compiler-plugin</artifactId>
 				<version>3.10.0</version>
 				<configuration>
-					<source>1.6</source>
-					<target>1.6</target>
+					<source>1.8</source>
```

Comment:
I see why you prefer to set this to a value which allows to build with a current JDK. But we must stay on 1.6 here for compatibility with pdfbox-2.0.